web/admin.py

```diff
@@ -27,10 +27,22 @@
 
 from django.contrib import admin
 
-from web.models import SwimRecord
+from .models import SwimRecord, Meet, MeetDocument
 
 
 class SwimRecordAdmin(admin.ModelAdmin):
     list_filter = ['record_type']
 
+
+class MeetDocumentInline(admin.StackedInline):
```

Comment:
StackedInline is used to manage one-to-many relationships, not to enforce one-to-one relationships. See my comment about the models.